config/packages/security.yaml

```diff
@@ -1,13 +1,31 @@
 security:
+    role_hierarchy:
+        ROLE_UTILISATEUR: ROLE_USER
+        ROLE_RESPONSABLE:  ROLE_UTILISATEUR
```

Comment:
What's this strange role hierarchy ?
do you really need a ROLE_UTILISATEUR and a ROLE_USER ?!?
You really should use english for your code.

Reply:
Deleted AppFixtures.php which is indeed useless
Added "Vous êtes connnecté en tant que X" to base.html.twig
Renamed ROLES